code/game/objects/items/broom.dm

```diff
@@ -0,0 +1,66 @@
+/obj/item/broom
+	name = "broom"
+	desc = "This is my BROOMSTICK! It can be used manually or braced with two hands to sweep items as you move. It has a telescopic handle for compact storage."
+	icon = 'icons/obj/janitor.dmi'
+	icon_state = "broom0"
+	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/custodial_righthand.dmi'
+	force = 8
+	throwforce = 10
+	throw_speed = 3
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
+	attack_verb = list("swept", "brushed off", "bludgeoned", "whacked")
+	resistance_flags = FLAMMABLE
+	var/wielded = FALSE // track wielded status on item
```

Comment:
dont do this

Reply:
NO

Reply:
Based on Ninja's comment to track the wieled status using signals this is the action that is taken.

Reply:
you can just use a signal in the component which works more generalized.

Reply:
Also this one is resolved, this is not on broom now.

Reply:
it still is, it needs to be in the component as it's very generalized; it'll be nice for someone who makes a new item which only has a different icon for 2hing